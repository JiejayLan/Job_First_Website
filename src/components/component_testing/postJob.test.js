import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import PostJob from "../business_profile/postJob"
//import "../../style/postJob.css";

describe("postJob snapshot test", () => {
    test('postJob render should match snapshot', () => {
        const render = new ReactShallowRenderer();
        render.render(<PostJob />);
        expect(render.getRenderOutput()).toMatchSnapshot();
    });
});