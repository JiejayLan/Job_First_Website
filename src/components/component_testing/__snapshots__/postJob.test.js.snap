// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postJob snapshot test postJob render should match snapshot 1`] = `
<div align="center">
            <br />
            <h1>{I18n.get('Post a New Job')}</h1>
            <Form onSubmit="*" className="main-form" style={{ "width": "80%" }}>
                <Form.Item>
                    <Input placeholder={I18n.get('Enter Employer Name')}
                        prefix={<Icon type="user" />}
                        suffix={
                            <Tooltip title={I18n.get('Enter the name of the employer.')}>
                                <Icon type="info-circle" />
                            </Tooltip>}
                    />
                </Form.Item>
                <Form.Item>
                    <DatePicker placeholder={I18n.get('Date Posted On')} />
                </Form.Item>
                <Form.Item>
                    <Select placeholder={I18n.get('Job Type')}>
                        <Option value="Full Time">{I18n.get('Full Time')}</Option>
                        <Option value="Part Time">{I18n.get('Part Time')}</Option>
                        <Option value="Internship">{I18n.get('Internship')}</Option>
                        <Option value="Temporary">{I18n.get('Temporary')}</Option>
                    </Select>
                </Form.Item>
                <Form.Item>
                    <TextArea
                        placeholder={I18n.get('Enter Job Description')} autosize={{ minRows: 2, maxRows: 6 }}
                    />
                </Form.Item>
                <Form.Item>
                    <Input placeholder="(###) ###-####"
                        prefix={<Icon type="contacts" />}
                        suffix={
                            <Tooltip title={I18n.get('Enter the contact number of the employer.')}>
                                <Icon type="info-circle" />
                            </Tooltip>}
                    />
                </Form.Item>
                <Form.Item>
                    <Button type="primary" htmlType="submit">{I18n.get('Submit Job')}</Button>
                </Form.Item>
            </Form>
        </div>
`;
